### ISSUE TITLE
custom fonts don't work

### ISSUE THREAD

**Issue by sunaku** [pelya/xserver-xsdl#1]
Hello,

I'm using XServer XSDL version 1.11.07.

Adding fonts with `xset +fp` does not work.

Only the "fixed" font is available in `xfontsel`.

Does X font server need to be implemented?

Thanks for your consideration.

**Comment by pelya** [comment]
Xserver does not come with it's own fonts, clients usually provide their
own fonts, and don't depend on server-side fonts.
If you wish to have server-side fonts - copy directory /usr/share/fonts/X11
from your Linux to SD card on your Android device,
launch X server, tap button "Change device config" when it starts, go to
"Data install location" -> "Specify directory" , don't change path in the
first dialog,
in second dialog change command line to this (note that there are no spaces
between commas):
XSDL -fp
/sdcard/fonts/100dpi/,/sdcard/fonts/75dpi/,/sdcard/fonts/Type1/,/sdcard/fonts/misc/

**Comment by sunaku** [comment]
Thanks!  That fixed the problem for me. :smiley_cat: And it gave me a hint on making `xset +fp` work:

Previously, I was using `xset +fp ~/.fonts` and it failed because XSDL is running outside my Linux chroot.  In particular, XSDL has a different idea of what the home directory is and what `~/.fonts` resolves to because it's looking in Android filesystem instead of my Linux chroot filesystem.

Now I am using `xset +fp /sdcard/MyLinuxHomeDir/.fonts` from my Linux chroot and it works correctly because XSDL is able to resolve the path since it's specified in terms of Android filesystem instead of my Linux chroot filesystem. :neckbeard:

**Comment by rctan** [comment]
Hi, I followed your instruction to set the font path but when I performed an "xlsfonts -display [NEXUS 7 IP Address]:0" on the Linux PC, it did not see the new fonts. It listed these six fonts:

[root@DRSNexusTabletServer02 ~]# xlsfonts -display 192.168.42.129:0
-misc-fixed-medium-r-semicondensed--0-0-75-75-c-0-iso8859-1
-misc-fixed-medium-r-semicondensed--13-100-100-100-c-60-iso8859-1
-misc-fixed-medium-r-semicondensed--13-120-75-75-c-60-iso8859-1
6x13
cursor
fixed

I copied two fonts (10x20.pcf, 8x13.pcf) to the /sdcard/fonts folder. Any ideas why the X Server is not loading the fonts?  Thanks in advance!
